docs/userguide/networking/configure-dns.md

```diff
@@ -0,0 +1,118 @@
+<!--[metadata]>
++++
+title = "Configure container DNS in user-defined networks"
+description = "Learn how to configure DNS in user-defined networks"
+keywords = ["docker, DNS, network"]
+[menu.main]
+parent = "smn_networking"
++++
+<![end-metadata]-->
+
+# Embedded DNS server in user-defined networks
+
+The information in this section covers the embedded DNS server operation for containers in user-defined networks. DNS lookup for containers connected to user-defined networks works differently compared to the containers connected to `default bridge` network.
+
+**Note**: In order to maintain backward compatibility, the DNS configurations in `default bridge` network is retained with no behaviourial changes. Please refer to the [DNS in default bridge network](default_network/configure-dns.md) for more information on DNS configurations in `default bridge` network.
+
+As of v1.10, the docker daemon implements an embedded DNS server which provides built-in service discovery for any container created with a valid `name` or `net-alias` or aliased by `link`. The exact details of how Docker manages the DNS configurations inside the container can change from one Docker version to the next. So you should not assume the way the files such as `/etc/hosts`, `/etc/resolv.conf` are managed inside the containers and leave the files alone and use the following Docker options instead.
+
+Various options that affect container domain name services.
+
+<table>
```

Comment:
am not a doc-expert :)
am simply using the recommended approach in other documents :)